packages/protocol/socket.ts

```diff
@@ -33,19 +33,19 @@ let gNextMessageId = 1;
 
 export type CommandResponse =
   | {
-      id: number;
-      result: unknown;
-      error?: undefined;
-    }
+    id: number;
+    result: unknown;
+    error?: undefined;
+  }
```

Comment:
I'm not sure what's going on here, seems like somehow your prettier and trunk's prettier disagree?

Reply:
yeah - i think that's true.